app/javascript/components/HelloWorld.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+class HelloWorld extends React.Component {
+  render () {
+    return (
+      <React.Fragment>
+        <div>
+          Greeting: {this.props.greeting}
+        </div>
+      </React.Fragment>
+    );
+  }
+}
+
+HelloWorld.propTypes = {
+  greeting: PropTypes.string
+};
+export default HelloWorld
```

Comment:
For this HelloWorld.js and HelloWorld.tsx, we will just rename them later when we implement the react app?